os_migrate/plugins/module_utils/subnet.py

```diff
@@ -71,6 +71,13 @@ def _find_sdk_res(conn, name_or_id):
 
     @staticmethod
     def _update_sdk_res(conn, name_or_id, sdk_params):
+        # intermittently the subnet update fails when trying to update with a
+        # message that states the ip_version or network_id attributes cannot
+        # be updated.  See #140 for more information.
+        blacklist = ['network_id', 'project_id']
```

Comment:
Is it worth doing this in Resource class? Then i wouldn't then call it `blacklist`, but perhaps `readonly_sdk_params`. And we could call a method which will filter those params out from `sdk_params` right before calling _update_sdk_res here:

https://github.com/os-migrate/os-migrate/blob/master/os_migrate/plugins/module_utils/resource.py#L175

WDYT?

Reply:
We can also merge this fix first and then do the Resource subsequently if you want. And track it in a separate issue/card?

Reply:
I don't have a problem promoting this change into the base class.  I'll push an update to this patch in a few minutes.